supplements/glory-of-the-giants/barbarian-giant.xml

```diff
@@ -0,0 +1,104 @@
+		</sheet>
+	</element>
+
+	<element name="Elemental Cleaver" type="Archetype Feature" source="Bigby Presents: Glory of the Giants" id="ID_WOTC_GOTG_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_ELEMENTAL_CLEAVER">
+		<description>
+			<p><em>6th-level Path of the Giant feature</em></p>
+			<p>Your bond with the elemental might of giants grows, and you learn to infuse weapons with primordial energy.</p>
+			<p class="indent">When you enter your rage, you can choose one weapon that you are holding and infuse it with one of the following damage types: acid, cold, fire, thunder, or lightning. While you wield the infused weapon during your rage, the weapon's damage type changes to the chosen type, it deals an extra 1d6 damage of the chosen type when it hits, and it gains the thrown property, with a normal range of 20 feet and a long range of 60 feet. If you throw the weapon, it reappears in your hand the instant after it hits or misses a target. The infused weapon's benefits are suppressed while a creature other than you wields it.</p>
+			<p class="indent">While raging and holding the infused weapon, you can use a bonus action to change the infused weapon's current damage type to another one from the damage type options above.</p>
+		</description>
+		<sheet>
+			<description>When you enter your rage, you can choose one weapon that you are holding and infuse it with one of the following damage types: acid, cold, fire, thunder, or lightning. While you wield the infused weapon during your rage, the weapon's damage type changes to the chosen type, it deals an extra {{elemental cleaver:dice}}d6 damage of the chosen type, and it gains the thrown property, with a normal range of 20 feet and a long range of 60 feet. If you throw the weapon, it reappears in your hand the instant after it hits or misses a target. &#13;
+			While raging and holding the infused weapon, you can use a bonus action to change the infused weapon's current damage type to another one from the damage type options above.</description>
+		</sheet>
+		<rules>
+			<stat name="elemental cleaver:dice" value="1" bonus="base" />
+		</rules>
+	</element>
+
+	<element name="Mighty Impel" type="Archetype Feature" source="Bigby Presents: Glory of the Giants" id="ID_WOTC_GOTG_ARCHETYPE_FEATURE_PATH_OF_THE_GIANT_MIGHTY_IMPEL">
+		<description>
+			<p><em>10th-level Path of the Giant feature</em></p>
+			<p>Your connection to giant strength allows you to hurl both allies and enemies on the battlefield. As a bonus action while raging, you can choose one Medium or smaller creature within your reach and move it to an unoccupied space you can see within 30 feet of yourself. An unwilling creature must succeed on a Strength saving throw (DC equals 8 + your proficiency bonus + your Strength modifier) to avoid the effect.</p>
```

Comment:
This feature sounds terrible since they used generic "move" and in the next line use the specific "throw". 
I hope that if this is true to the book that they fix that in first errata